### DISCUSSION TITLE
Multiple checkboxes with the same wire:model , check one to select all

### DISCUSSION THREAD

**Discussion by Jamil-NE20** [livewire/livewire#5313] [1. Help]
Code: 

... Component {
   public $plan_requirement2 = [];
...
}

View:

<div class="row">                                          
             
                     <label for="requirement2">                                                        
                                <input type="checkbox" id = "requirement2" value="Red car" wire:model="plan_requirement2"> Car 1                                    
                       </label>
                        <label for="requirement2">
                                  <input type="checkbox" id = "requirement2" value="Blue car" wire:model="plan_requirement2"> Car 2
                         </label>                                              
               
 </div> 

Result view after check  [x ]Car 1:  checked both

 [ x]Car 1  [x ]Car 2

**Comment by joshhanley** [comment]
@Jamil-NE20 it looks like you are wire modelling to the same property, hence it will be overwriting the array. You should create array keys for each one to use like `wire:model="plan_requirement2.blue_car"`.

Hope this helps!
